packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -299,7 +304,8 @@ export const Tabs = (props: TabsProps) => {
           `iui-${type}`,
           {
             'iui-green': color === 'green',
-            'iui-animated': type !== 'default',
+            [isAnimated ? 'iui-animated' : 'iui-not-animated']:
+              type !== 'default',
```

Comment:
this can be simplified like this (make sure to test it first):
```suggestion
            'iui-animated': type !== 'default' && isClient,
            'iui-not-animated': type !== 'default' && !isClient,
```

Reply:
Thanks for the suggestion. I was stuck for a while on this thinking how to not make it look so tacky 🤣 Adjusted